javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,44 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (studentName === "") {
+    console.log("Not possible to add an empty name!");
+    return;
+  }
+
+  if (class07Students.includes(studentName)) {
+    console.log("This student" + studentName + "already exists");
```

Comment:
It seems like you haven't tested your code if it works

```suggestion
    console.log("This student " + studentName + " already exists");
```